www/significant-events-styles-sepia.css

```diff
@@ -1,10 +1,10 @@
 
 div#significant-changes-container{
-    background-color:#F8F1E3;
+    background-color:#E1DAD1;
```

Comment:
Just wanted to confirm the background colors on these buttons - they seemed a bit more divergent than what was is on other colors.
![Screenshot](https://user-images.githubusercontent.com/9295855/97212612-e1b4b780-177d-11eb-859f-36724c8cbfb7.png)

Reply:
Pretty sure this came up in our polish and @carolynlimadeo was good with it. She also might be rethinking them as a part of [T266354](https://phabricator.wikimedia.org/T266354). I'll ask her to confirm for this PR though.

Reply:
@mcleinman is that white snippet text a separate bug you noticed?

Reply:
I couldn't replicate. I assume it's related to going between themes. But I tried a bunch, and couldn't replicate it. (Meant to use another screen shot that didn't show it, but put the wrong one up there.)

Reply:
Hi all! Yeah, I need to redefine the button colors, but this color will work for now for user testing. Sorry about the confusion!